test/rbus/provider/rbusTestProvider.c

```diff
@@ -311,9 +311,9 @@ rbusError_t createTableRowNode(Node* parent, char const* alias, uint32_t* instNu
 
             assert(child->type == RBUS_ELEMENT_TYPE_TABLE_ROW);
```

Comment:
```child``` is de-referenced without checking its NULL or not.

Reply:
made changes and updated PR
but can we replace line 312 with assert(child !=NULL); i.e remove assert(child->type == RBUS_ELEMENT_TYPE_TABLE_ROW);

Since below line 314 is already handling this condition by calling rtListItem_GetNext() if (child->type == RBUS_ELEMENT_TYPE_TABLE_ROW) fails